src/OpenTelemetry.AutoInstrumentation.StartupHook/RulesEngine/ApplicationInExcludeListRule.cs

```diff
@@ -55,12 +70,18 @@ private static string GetApplicationName()
         }
     }
 
-    private static bool IsApplicationInExcludeList(string applicationName)
+    private static bool IsApplicationInExcludeList(string appDomainName, string processModuleName)
+    {
+        return GetExcludedApplicationNames().Contains(processModuleName, StringComparer.InvariantCultureIgnoreCase) ||
+            GetExcludedAppDomainNames().Contains(appDomainName, StringComparer.InvariantCultureIgnoreCase);
+    }
+
+    private static ICollection<string> GetExcludedAppDomainNames()
     {
-        return GetExcludedApplicationNames().Contains(applicationName);
+        return new[] { "dotnet", "dotnet.exe" };
```

Comment:
Since this is the domain name I wasn't expecting `dotnet.exe` even for .NET Framework apps. I'm pretty positive that it will be `dotnet` for .NET in general.

It doesn't matter much in this case, but, why not a local instead? Anyway, if we can have only `dotnet` this will be a mute point.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2765/commits/153066b1fd3b6ba63c28d5912dad6ced32cd54db

Most of the time for me it's better readability which hopefully compiler will inline anyway.